arm-sql/compositeSql.json

```diff
@@ -13,8 +13,6 @@
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/databaseSecurityAlertPolicies.json",
-    "./2015-05-01-preview/swagger/blobAuditingPolicies.json",
```

Comment:
Are you merging these into a single swagger? If not, are ```failoverGroups.json``` or ```virtualNetworkRules.json``` going to be used anywhere?
@vishrutshah will this affect the Ruby builds?